src/strings/index.tsx

```diff
@@ -868,6 +868,10 @@ const strings = new LocalizedStrings({
     NUMBER_OF_PLANTS_BY_SPECIES: 'Number of Plants by Species',
     NUMBER_OF_PLANTS_BY_PLOT_AND_SPECIES: 'Number of Plants by Plot and Species',
     NO_PLANTS: 'No Plants',
+    NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_TITLE: 'No Seedlings Available to Outplant',
+    NO_SEEDLINGS_AVAILABLE_TO_OUTPLANT_DESCRIPTION:
+      'There are no batches selected with seedlings that are "ready". Select another withdraw purpose or cancel and select a different batch.',
```

Comment:
This is very minor, but for messaging, I think the period goes inside the quotation mark (https://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html).

`"ready."` instead of `"ready".`

Similarly for line 874 below.